src/borg/remote.py

```diff
@@ -647,7 +647,6 @@ def borg_cmd(self, args, testing):
         # give some args/options to 'borg serve' process as they were given to us
         opts = []
         if args is not None:
-            opts.append('--umask=%03o' % args.umask)
```

Comment:
a new client (using this code) would not send `--umask=XXX` to the server.

an old server would just use the default umask (077) in that case.